src/common/utils.ts

```diff
@@ -10,6 +10,10 @@ const REMOVE_SPECIAL_CHARACTERS_REGEX = /[^a-zA-Z0-9-_#.]/g;
 
 export const DELETED_BY_WORKER = 'worker';
 
+export const playwrightUser = {
+  id: '8bf2UpTsHFnczzOlk0mtg',
+};
```

Comment:
you can add him to `isSpecialUser` as well

Reply:
Mostly used for njord now right?
I think the only issue for now was the random popups it would get.

Reply:
we use it on few places, to ignore those kind of special users, but yeah no blocker